components/public-api/gitpod/experimental/v2/organization.proto

```diff
@@ -0,0 +1,208 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
+
+import "google/protobuf/timestamp.proto";
+import "gitpod/experimental/v2/pagination.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v2";
+
+
+message Organization {
+    string id = 1;
+    string name = 2;
+    google.protobuf.Timestamp creation_time = 3;
+}
+
+message OrganizationMember {
+    string user_id = 1;
+    OrganizationRole role = 2;
+    google.protobuf.Timestamp member_since = 3;
+    optional string avatar_url = 4;
+    optional string full_name = 5;
+    optional string primary_email = 6;
```

Comment:
Where are we using this? Is it possible to rename this field to sth else, e.g. `email`? (Not sure what it's value is)

The reason I'm asking is [this issue](https://linear.app/gitpod/issue/EXP-202/persist-primary-email-instead-of-guessing), basically, and the confusing semantics of `primary`.